packages/api/src/controllers/taskController.js

```diff
@@ -599,23 +621,48 @@ async function patchManagementPlanStartDate(trx, req, typeOfTask, task = req.bod
   }
 }
 
-async function notifyAssignee(userId, taskId, taskTranslationKey, farmId) {
-  if (!userId) return;
+async function getTaskStatus(taskId) {
+  return await TaskModel.query()
```

Comment:
We need to be sure that all queries consider [soft deletes](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1184235521/Navigating+and+querying+the+LiteFarm+database#A-note-on-%E2%80%98deleted%E2%80%99) which I will describe at tomorrow's stand up.

Reply:
@seinnlee doesn't seem like this was addressed. Add a delete: false to the where statement.